cli/import-jdl.js

```diff
@@ -153,6 +153,9 @@ const generateApplicationFiles = ({ generator, application, withEntities, inFold
     const cwd = inFolder ? path.join(generator.pwd, baseName) : generator.pwd;
     logger.debug(`Child process will be triggered for ${runYeomanProcess} with cwd: ${cwd}`);
 
+    if (process.argv.includes('--no-insight')) {
+        generator.options.noInsight = true;
```

Comment:
This should happen automatically, isn't it working? let me know if it doesnt work.

Reply:
No it doesn't work few days ago. I didn't test with your last big change, I can test right now

Reply:
I confirm. It doesn't work.
See my logs:
```
jhipster@a548dbae8523:~/app$ jhipster import-jdl *.jdl --no-insight
INFO! Using JHipster version installed globally
INFO! Executing import-jdl app.jdl
INFO! Options: insight: false, from-cli: true
INFO! The JDL is being parsed.
warn: An Entity name 'User' was used: 'User' is an entity created by default by JHipster. All relationships toward it will be kept but any attributes and relationships from it will be disregarded.
info: JPAMetaModelFiltering has been detected for BankAccount, the 'serviceClass' value for the 'service' is gonna be set for this entity if no other value has been set.
INFO! Found entities: BankAccount, TheLabel, Operation.
INFO! The JDL has been successfully parsed
INFO! Generating 1 application.
INFO! Generating application jhipsterSampleApplication in a new parallel process
INFO! Executing jhipster:app on /home/jhipster/app
INFO! Options: fromCli: true, withEntities: true, force: true, from-cli: true, with-entities: true


        ██╗ ██╗   ██╗ ████████╗ ███████╗   ██████╗ ████████╗ ████████╗ ███████╗
        ██║ ██║   ██║ ╚══██╔══╝ ██╔═══██╗ ██╔════╝ ╚══██╔══╝ ██╔═════╝ ██╔═══██╗
        ██║ ████████║    ██║    ███████╔╝ ╚█████╗     ██║    ██████╗   ███████╔╝
  ██╗   ██║ ██╔═══██║    ██║    ██╔════╝   ╚═══██╗    ██║    ██╔═══╝   ██╔══██║
  ╚██████╔╝ ██║   ██║ ████████╗ ██║       ██████╔╝    ██║    ████████╗ ██║  ╚██╗
   ╚═════╝  ╚═╝   ╚═╝ ╚═══════╝ ╚═╝       ╚═════╝     ╚═╝    ╚═══════╝ ╚═╝   ╚═╝

                            https://www.jhipster.tech

Welcome to JHipster v5.6.1
Application files will be generated in folder: /home/jhipster/app
 _______________________________________________________________________________________________________________

  Documentation for creating an application is at https://www.jhipster.tech/creating-an-app/
  If you find JHipster useful, consider sponsoring the project at https://opencollective.com/generator-jhipster
 _______________________________________________________________________________________________________________

? May JHipster anonymously report usage statistics to improve the tool over time? (Y/n)
```

Reply:
let me take a look, NPM tests are passing without it though

Reply:
Thanks @deepu105 
But no worry, it is not needed immediatly, it can wait few days / weeks. It's for improving our CI, so we'll be able to use JDL

Reply:
@pascalgrimaud I pushed a fix, it was bug in option parsing. Let me know if it works now